pkg/lca/imagebasedupgrade.go

```diff
@@ -223,6 +223,22 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
 	return builder
 }
 
+// WithAdditionalImages adds additionalImages to be used by the imagebasedupgrade.
+func (builder *ImageBasedUpgradeBuilder) WithAdditionalImages(
+	additionalImagesConfigMapName, additionalImagesConfigMapNamespace string) *ImageBasedUpgradeBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Appending additionalImages configmap name %s in namespace %s to the imagebasedupgrade",
```

Comment:
Just one technicality 
```suggestion
	glog.V(100).Infof("Adding additionalImages configmap name %s in namespace %s to the imagebasedupgrade",
```
Appending make it seem that it is part of a slice. But it is really just one configmap

Reply:
You're right.
Thanks.